lib/usd/translators/shading/CMakeLists.txt

```diff
@@ -23,6 +23,14 @@ target_sources(${TARGET_NAME}
         usdStandardSurfaceWriter.cpp
 )
 
+if (MAYA_API_VERSION VERSION_GREATER_EQUAL 20250300)
+    target_sources(${TARGET_NAME}
+        PRIVATE
+            usdOpenPBRSurfaceWriter.cpp
+            usdOpenPBRSurfaceReader.cpp
```

Comment:
Can export to UsdPreviewSurface as long as Maya has a native OpenPBR Surface node.